app/controllers/queries.js

```diff
@@ -73,3 +110,37 @@ module.exports = {
     }
   },
 };
+
+const getEligibleResponders = (sellers) => {
+  try {
+    let eligibleResponders = {};
+    let wired_up_users = [];
+    let non_wired_up_users = [];
+
+    sellers.buyer_selected_sellers.forEach((item) => {
+      // in buyer_selected_sellers we have uuid's so we can push to tempSellers[wired_up_users]
+      wired_up_users.push(item);
+    });
+
+    sellers.system_selected_sellers.forEach((item) => {
+      // in buyer_selected_sellers we have uuid's so we can push to tempSellers[wired_up_users]
+      wired_up_users.push(item);
+    });
+
+    sellers.addressbook_contacts.forEach((item) => {
+      // in addressbooks contact some contact will be wiredup users and some won't
+      if (item.mobile && item.email) {
+        non_wired_up_users.push(item);
+      } else {
+        wired_up_users.push(item);
```

Comment:
@rajeshn95 Do we assume we already have user UUID in this case? That frontend is pushing uuid along with addressbook_contacts.

If address book contact has - item.mobile && item.email, then there is a chance they might be wiredup user actually. The conditions doesn't seem correct here.

Reply:
payload for create query: 
```
{ 
    "type": "refinance_existing_loan",
    "data": {
        "initial": {
            "refinance_details":{ 
                "loan_currency": "$",
                "outstanding_loan_amount": "10000000",
                "loan_type": "", 
                "loan_taken_date": "27/09/2021", 
                "maturity_date": "27/09/2022", 
                "interest_rate": "10%", 
                "funding_bank": "SBI", 
                "attachments": "link",
                "remarks_for_the_seller": "helloo test test test test"
            },
            "product_details":{ 
                "product": "Vendor Financing",
                "product_special_request": "something special requested" 
            }
        },
        "expiry":{
            "response_restriction": 10,
            "query_valid_till": 4 
        }
    },
    "sellers":{
        "addressbook_contacts":[
            {
                "email": "abc@mail.com",
                "mobile": "1234567890"
            },
            {
                "email": "xyz@mail.com",
                "mobile": "9876543210"
            },
            "7331ba28-e75c-489a-ab4b-d3210f20f760"
        ],
        "buyer_selected_sellers":[
            "7331ba28-e75c-489a-ab4b-d3210f20f763", 
            "7331ba28-e75c-489a-ab4b-d3210f20f764"
        ],
        "system_selected_sellers":[
            "7331ba28-e75c-489a-ab4b-d3210f20f765", 
            "7331ba28-e75c-489a-ab4b-d3210f20f766"
        ]
    },
    "status": "open"
}
```